tests/test_behavior/test_audio_interface.py

```diff
@@ -89,8 +90,14 @@ def test_get_metadata(self):
         self.assertEqual(len(audio_metadata), self.num_audio_files)
 
     def test_incorrect_write_as(self):
-        expected_error_message = "Acoustic series can be written either as 'stimulus' or 'acquisition'."
-        with self.assertRaisesWith(exc_type=AssertionError, exc_msg=expected_error_message):
+        expected_error_message = """'test' is not one of ['stimulus', 'acquisition']
+
+Failed validating 'enum' in schema['properties']['Audio']['properties']['write_as']:
+    {'default': 'stimulus', 'enum': ['stimulus', 'acquisition']}
+
+On instance['Audio']['write_as']:
+    'test'"""
+        with self.assertRaisesWith(exc_type=jsonschema.exceptions.ValidationError, exc_msg=expected_error_message):
```

Comment:
I am taking the Literal values and turning them into "enum" entries in the json-schema, so if you try to enter a value that is not one of these, it will be caught by the json schema validation.